src/microbe_stage/MicrobeSpecies.cs

```diff
@@ -51,6 +53,30 @@ public void SetInitialCompoundsForChemo()
         InitialCompounds.Add("hydrogensulfide", 10);
     }
 
+    public void SetUpdatedCompounds()
+    {
+        var simulation = SimulationParameters.Instance;
+
+        // If you have iron (f is the symbol for rusticyanin)
```

Comment:
I don't think this comment is accurate or really adds anything, should be just removed.